ports/libidn2/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "libidn2",
   "version-string": "2.3.0",
```

```suggestion
  "version": "2.3.0",
```
We prefer to use more proper field here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md.